### PR TITLE
[INJIMOB-932]  - PixelPass implementation

### PR DESCRIPTION
**Pull request by sree96**

### CODE REVIEW

File: screens/Scan/ScanScreen.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {isIOS} from '../../shared/constants';
 import {BannerNotificationContainer} from '../../components/BannerNotificationContainer';
 import {SharingStatusModal} from './SharingStatusModal';
 import {SvgImage} from '../../components/ui/svg';
+import {assign} from "xstate";
```
Comment (Alka1703) [inline screens/Scan/ScanScreen.tsx:18]: This import is not getting used it seems. If added intentionally we should ideally avoid using xstate in UI components.

Comment (sree96) [reply screens/Scan/ScanScreen.tsx]: It was added for some other component I guess. Its not used anyways. Removed it. Plz check

File: components/BannerNotificationContainer.tsx [hunk 2]
```diff
@@ -29,6 +34,18 @@ export const BannerNotificationContainer: React.FC = () => {
         </View>
       )}
 
+      {showQuickShareSuccessBanner && (
+          <View style={{marginTop: 10, marginBottom: 10}}>
```
Comment (swatigoel) [inline components/BannerNotificationContainer.tsx:38]: Can we move this style to style file

File: components/BannerNotificationContainer.tsx [hunk 3]
```diff
@@ -29,6 +34,18 @@ export const BannerNotificationContainer: React.FC = () => {
...
+            <BannerNotification
+                type={BANNER_TYPE_SUCCESS}
+                message={rt('status.accepted.message')}
+                onClosePress={scanScreenController.DISMISS_QUICK_SHARE_BANNER}
+                key={'activatedVcPopup'}
+                testId={'activatedVcPopup'}
+            />
```
Comment (swatigoel) [inline components/BannerNotificationContainer.tsx:45]: Why key and testId are activatedVcPopup?

File: components/QrCodeOverlay.tsx [hunk 4]
```diff
@@ -7,69 +7,103 @@ import {Theme} from './ui/styleUtils';
...
+    }
+    return qrData;
+  }
 
+  useEffect(() => {
+    (async () => {
+      const qrString = await getQRData();
+      setQrString(qrString);
+    })();
+  }, []);
```
Comment (swatigoel) [inline components/QrCodeOverlay.tsx:43]: There is no dependency in useEffect, do we want to reload this every time?
Every change due to useEffect will cause component re-render

File: components/QrCodeOverlay.tsx [hunk 5]
```diff
@@ -7,69 +7,103 @@ import {Theme} from './ui/styleUtils';
...
-            <Icon
-              {...testIDProps('qrCodeCloseIcon')}
-              name="close"
-              onPress={toggleQrOverlay}
-              color={Theme.Colors.Details}
-              size={32}
-            />
-          </Row>
-          <Centered testID="qrCodeDetails" pY={30}>
+    qrString != '' && (
```
Comment (swatigoel) [inline components/QrCodeOverlay.tsx:47]: If qrString is empty, are we expecting to show something to user?
Or we expect qrString will never be empty then we don't need this check

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/QrCodeOverlay.tsx (hunk 5 above)
Comment (sree96) [reply components/QrCodeOverlay.tsx:47]: There was a discussion around this to show some error icon if the QR could not be generated.

Comment (sree96) [reply components/QrCodeOverlay.tsx:47]: I think we need UI for same.

File: components/BannerNotificationContainer.tsx (hunk 2 above)
Comment (sree96) [reply components/BannerNotificationContainer.tsx]: Updated plz take a look

File: components/BannerNotificationContainer.tsx (hunk 3 above)
Comment (sree96) [reply components/BannerNotificationContainer.tsx:46]: Updated plz take a look

File: components/QrCodeOverlay.tsx (hunk 4 above)
Comment (sree96) [reply components/QrCodeOverlay.tsx]: Updated plz take a look

**Review by swatigoel — APPROVED** [review]

**Pull request merged**